exercises/protein-translation/examples/success-standard/src/ProteinTranslation.hs

```diff
@@ -0,0 +1,26 @@
+module ProteinTranslation (toProtein) where
+
+import Data.List.Split (chunksOf)
+
+toProtein :: String -> [String]
+toProtein strand = takeWhile ("STOP" /=) $ map codonToProtein $ chunksOf 3 strand
+
+codonToProtein :: String -> String
```

Comment:
Just a quick question, maybe I'm missing something, but to me the description is a little misleading. It seems that the correct solution would be just to do the map operation written in the description, but judging by this and the test cases it seems that it actually expects the protein names instead of the codons?

Reply:
I was basing this off the Scala version: http://exercism.io/exercises/scala/protein-translation/test-suite